spring-cloud-modules/pom.xml

```diff
@@ -19,7 +19,7 @@
     <modules>
         <module>spring-cloud-loadbalancer</module>
         <module>spring-cloud-config</module>
-        <module>spring-cloud-eureka</module>
+<!--        <module>spring-cloud-eureka</module>-->
```

Comment:
mention here that "uncomment when spring-cloud-modules is moved to JDK9+ profile"